static/app/views/replays/replayTable.tsx

```diff
@@ -142,15 +142,16 @@ function ReplayTableRow({
           <Link
             to={`/organizations/${organization.slug}/replays/${project?.slug}:${replay.id}/`}
           >
-            {replay.user.username ||
-              replay.user.name ||
-              replay.user.email ||
-              replay.user.ip_address ||
-              replay.user.id ||
-              ''}
+            {replay.user.displayName || ''}
           </Link>
         }
-        user={replay.user}
+        user={{
+          username: replay.user.displayName || '',
+          email: replay.user.email || '',
+          id: replay.user.id || '',
+          ip_address: replay.user.ip_address || '',
+          name: replay.user.name || '',
+        }}
```

Comment:
This feeds into the UserAvatar component, so we've got to marshal the data to fit that type.

Reply:
Can you suggest a specific code change for me?

Replays don't have a username field, so that's really whats getting filled in. replay.user.displayName comes from:
https://github.com/getsentry/sentry/commit/065bfb1dca636a91f23cb9a652603669840a762d
```
        item["user"]["displayName"] = (
            item["user"]["name"]
            or item["user"]["email"]
            or item["user"]["ip_address"]
            or item["user"]["id"]
        )
```
so to me it feels like out version of `def get_display_name(self): ` will be about the same: `return name | email | username [name | email | ip_address | id] ]`

of course i didn't really think too hard about it earlier, just thinking and typing through it now.

Reply:
Sorry this is more of a comment for our backend api but `displayName` probably should become `name` and the previous `name` field to `username`

https://docs.sentry.io/platforms/javascript/enriching-events/identify-user/

Reply:
ok, i'll put this through and we can update the shape that the backend returns and that'll have knock on effects here.